src/Playroom/Toolbar/Toolbar.tsx

```diff
@@ -148,58 +160,56 @@ export default ({ themes: allThemes, widths: allWidths, snippets }: Props) => {
             </ToolbarItem>
           </div>
         </div>
-
-        <div className={styles.panel}>
-          {isSnippetsOpen && (
-            <div
-              hidden={isSnippetsOpen ? undefined : true}
-              className={styles.preference}
-            >
-              <Snippets
-                snippets={snippets}
-                onHighlight={(snippet) => {
-                  dispatch({
-                    type: 'previewSnippet',
-                    payload: { snippet },
-                  });
-                }}
-                onClose={(snippet) => {
-                  if (snippet) {
+        <CSSTransition
+          in={isOpen}
+          timeout={ANIMATION_TIMEOUT}
+          classNames={styles.transitionStyles}
+          mountOnEnter
+          unmountOnExit
+          onExited={() => setLastActivePanel(undefined)}
+        >
+          <div className={styles.panel} id="custom-id">
+            <div className={styles.preference}>
```

Comment:
These can be merged now that the inner `div` isn't doing anything other than applying a style.

In fact, is it worth just merging the `panel` style with the `preference` style?